charts/tidb-drainer/values.yaml

```diff
@@ -38,6 +43,12 @@ config: |
   [syncer.to]
   dir = "/data/pb"
 
+security: |
+  [security]
+  ssl-ca = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+  ssl-cert = "/var/lib/drainer-tls/cert.pem"
+  ssl-key = "/var/lib/drainer-tls/key.pem"
+
```

Comment:
Remove them from values.yaml since they cannot be changed.

Reply:
done